app/test/e2e/vocabulary-dataset.spec.js

```diff
@@ -0,0 +1,150 @@
+const nock = require('nock');
+const chai = require('chai');
+const Resource = require('models/resource.model');
+const Vocabulary = require('models/vocabulary.model');
+
+const { USERS } = require('./test.constants');
+const { mockDataset } = require('./utils');
+const { getTestServer } = require('./test-server');
+
+chai.should();
+
+let requester;
+
+nock.disableNetConnect();
+nock.enableNetConnect(process.env.HOST_IP);
+
+describe('Vocabulary-dataset relationships test suite', () => {
+    before(async () => {
+        if (process.env.NODE_ENV !== 'test') {
+            throw Error(`Running the test suite with NODE_ENV ${process.env.NODE_ENV} may result in permanent data loss. Please use NODE_ENV=test.`);
+        }
+
+        requester = await getTestServer();
+
+        Resource.remove({}).exec();
```

Comment:
- these return promises, prepend with await
- https://mongoosejs.com/docs/deprecations.html#remove

Reply:
Fixed in all tests of the repo 👍